projects/addon-editor/tokens/tiptap-editor.ts

```diff
@@ -0,0 +1,3 @@
+import {InjectionToken} from '@angular/core';
+
+export const TIPTAP_EDITOR = new InjectionToken('Token for Tiptap Editor');
```

Comment:
Can we lazy-load it similarly to YouTube API I did here?
https://medium.com/angularwave/lazy-loading-libs-from-cdn-in-angular-cdd5a6a32ed5?sk=a5180cdd92708118ff13d902cbfac7a5
Or with a `Promise` token as Fox did in the tweet mentioned in the article